test/images/serve_hostname/Makefile

```diff
@@ -31,7 +31,7 @@ ALL_ARCH = amd64 arm arm64 ppc64le s390x
 
 GOARM=7
 TEMP_DIR := $(shell mktemp -d)
-GOLANG_VERSION=1.7.5
+GOLANG_VERSION=1.8
```

Comment:
I think the 1.8 actually means "the latest version on the v1.8 release branch", but it makes sense to lock to a well-known version.

Will update